LanternExtractor/EQ/Wld/Exporters/ActorGltfExporter.cs

```diff
@@ -172,8 +172,17 @@ private static void ExportZone(WldFileZone wldFileZone, Settings settings, ILogg
             var textureImageFolder = $"{wldFileZone.GetExportFolderForWldType()}Textures/";
             gltfWriter.GenerateGltfMaterials(materialLists, textureImageFolder);
 
+            var bspNodes = wldFileZone.GetFragmentsOfType<BspTree>();
             foreach (var mesh in zoneMeshes)
             {
+                if (settings.ExportZoneMeshGroups)
```

Comment:
Looks like your PR is missing the corresponding settings.txt and Settings.cs changes.